src/DependencyInjection/Configuration.php

```diff
@@ -20,7 +20,7 @@
 final class Configuration implements ConfigurationInterface
 {
     /**
-     * @psalm-suppress PossiblyNullReference, PossiblyUndefinedMethod
+     * @psalm-suppress PossiblyNullReference, UndefinedInterfaceMethod, UnusedPsalmSuppress, PossiblyUndefinedMethod
```

Comment:
You dont need to superr UnusedPsalmSuppress, just remove the unused psalm supression instead. I think it is: PossiblyUndefinedMethod